### PR TITLE
Use content_status to populate the withdrawn boolean on course

### PR DESCRIPTION
**Pull request by davidgisbey**
## Context

In order to implement the designs for the new replace application choice flow. We need to know if a course option was withdrawn (the whole course is discontinued) or simply became full.

We can find this out by using FIND's API as the content status attribute is 'withdrawn' if a course has been withdrawn. 

## Changes proposed in this pull request

- Populate the withdrawn boolean on courses via the FIND V3 API.

## Link to Trello card

https://trello.com/c/kUjDNNbH/1646-dev-update-course-choices-if-course-becomes-full-while-waiting-for-references

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: spec/support/test_helpers/find_api_helper.rb [hunk 1]
```diff
@@ -139,7 +141,8 @@ def stub_find_api_provider_200_with_accredited_provider(
     start_date: Time.zone.local(2020, 10, 31),
     course_length: 'OneYear',
     region_code: 'north_west',
-    age_range_in_years: '4 to 8'
+    age_range_in_years: '4 to 8',
+    content_status: 'pubished'
```
Comment (malcolmbaig) [inline spec/support/test_helpers/find_api_helper.rb:145]: `pubished` -> `published`?

**Comment by malcolmbaig** [comment]
Nice find - is the intention for this to be used for another course option status check like https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/models/course_option.rb#L32 ?

File: spec/support/test_helpers/find_api_helper.rb (hunk 1 above)
Comment (davidgisbey) [reply spec/support/test_helpers/find_api_helper.rb]: thanks 👍

File: spec/support/test_helpers/find_api_helper.rb [hunk 2]
```diff
@@ -13,7 +13,8 @@ def stub_find_api_provider_200(
     site_address_line2: 'C/O The Bruntcliffe Academy',
     funding_type: 'fee',
     age_range_in_years: '4 to 8',
-    vac_status: 'full_time_vacancies'
+    vac_status: 'full_time_vacancies',
+    content_status: 'pubished'
```
[the remark this answers is not in the document]
Comment (davidgisbey) [reply spec/support/test_helpers/find_api_helper.rb]: Just a typo 🤦

**Comment by davidgisbey** [comment]
Thanks. I've added a test. Yes, it will Malcolm.

**Pull request merged**